### DISCUSSION TITLE
how to ignore a update from a user

### DISCUSSION THREAD

**Discussion by AminShahrabi** [kianahmadian/python-bale-bot#8] [Q&A] [answered]
so , i have a bot and i need to ignore updates from a user that i banned , is that possible?

**Comment by kian-ahmadian** [comment, marked as answer]
Hi @Codingwithamin

> **You can set a custom updater for do this.**

```py
from bale import Updater

class CustomUpdater(Updater):
def __init__(self, bot):
    super().__init__(bot)

def call_to_dispatch(self, update):
   if not ( update.type.is_message_update() and update.message.author.user_id == 1234):
       return await super().call_to_dispatch(update)
```
now you can set your custom updater to bot.
```
bot = bale.Bot(updater = CustomUpdater, ...)
```

Enjoy! 🤗

**Reply by okaeiz** [reply]
Hi there again! I am back again with a plenty of questions. I set a function to check whether the first message sent to the bot is "start" or "/start".
```python
"""start command handler"""
@client.listen(EventType.MESSAGE)
async def when_receive_message(message: Message):
    if message.text in ("/start", "start"):
     ...
    else:
     await message.reply("Please use the /start command.")
```
but after handling the first message, I want this condition to be ignored. Even after sending /start, if the users sends ANYTHING, the bot assumes the user should not be sending any messages other than "start" and "/start".

**Reply by kian-ahmadian** [reply]
**If I understand correctly, you want to know if this is the first time the user has messaged the bot or not.Well, you need to use the database to find out, and check the authors of the message.**

**Reply by okaeiz** [reply]
I don't necessarily need to know whether this is the first time anybody is sending the command to the bot or not. That's because a user may stop and start the bot several times.

When I ask the user to send a text message - their full name for example - the bot still assumes that the user should send "/start" or "start".

How can I quit that condition?

**Reply by kian-ahmadian** [reply]
### **This matter depends on you**
**but** for example, one of the tasks is to create this personalized updater for you. For example, you can put your users sending a special message into a variable, and then if they were sending a special message, you can connect them to another event.
